home/whatsapp.py

```diff
@@ -346,6 +346,47 @@ def create_standalone_template_header_components(
     return components
 
 
+def submit_to_meta_menu_action(model: Any) -> None:
+    from .models import WhatsAppTemplate
+
+    revision = model.get_latest_revision()
```

Comment:
This might be wrong, so we should check it, but I think in the list view where you do the bulk action, it shows the latest published version, but when you go in and look at the edit view, it shows the latest revision.

So if that's true, for bulk action, should we not be submitting the latest published version? And for the individual submission, we submit the latest revision?